path/_common/glob_to_reg_exp.ts

```diff
@@ -65,7 +65,7 @@ export function _globToRegExp(
 
   // Remove trailing separators.
   let newLength = glob.length;
-  for (; newLength > 1 && c.seps.includes(glob[newLength - 1]); newLength--);
+  for (; newLength > 1 && c.seps.includes(glob[newLength - 1]!); newLength--);
```

Comment:
`glob` is accessing their values using the `newLength` that is (initially) the length of `glob`. So, using `!` is justified as the elements accessed will always be defined